images/radicle-server/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM debian:stretch
+# The distribution must be the same one the binary is build with to
+# ensure binary compatibility. We use the `haskell` images on CI and
+# `fpco/stack-build` locally.
```

Comment:
What is this referring to? (I mean using `fpco/stack-build` locally.)

Reply:
This refers to building the binary with `stack build --docker`. I added that to the documentation.